sdk/ai/azure-ai-projects/azure/ai/projects/models/_patch.py

```diff
@@ -74,9 +73,11 @@
     MessageDeltaTextContent,
 )
 
+from ._models import ThreadMessage as ThreadMessageGenerated
+
 logger = logging.getLogger(__name__)
 
-StreamEventData = Union[MessageDeltaChunk, MessageDeltaTextContent, ThreadMessage, ThreadRun, RunStep, str]
+StreamEventData = Union["AgentMessageDeltaChunk", "AgentThreadMessage", ThreadRun, RunStep, str]
```

Comment:
Naming changes will impact to existing samples and customers. We shall avoid if no clear reason.

I think ThreadMessage is better than AgentThreadMessage, I do not see need to specify Agent there.

Reply:
Done